pkg/start/start.go

```diff
@@ -511,7 +517,7 @@ func (o *Options) NewControllerContext(cb *ClientBuilder, alwaysEnableCapabiliti
 		o.PromQLTarget,
 		o.InjectClusterIdIntoPromQL,
 		o.UpdateService,
-		alwaysEnableCapabilities,
+		stringsToCapabilities(o.AlwaysEnableCapabilities),
```

Comment:
This is the main point of the change - all other `o.Members` are passed into `NewControllerContext` via `Options`, and the method relies on earlier validation. Treating one `Options` member in a different way makes the interface messy.